### PR TITLE
Add tests and update docs for K06 and K07

### PR DESCRIPTION
**Pull request by christopher-davis-afs**
## Describe your changes

Per my understanding, this is all that's required to consider these two blocks implemented.

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] If OCPP 2.0.1: I have updated the [OCPP 2.0.1 status document](https://github.com/EVerest/libocpp/tree/main/doc/ocpp_201_status.md)
- [x] I read the [contribution documentation](https://github.com/EVerest/EVerest/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements

### CODE REVIEW

**Review by folkengine — APPROVED** [review]
The best kind of PR. All tests, no code.

File: doc/ocpp_201_status.md [hunk 1]
```diff
@@ -1266,7 +1266,7 @@ This document contains the status of which OCPP 2.0.1 numbered functional requir
 | K01.FR.40 | ✅     | `Absolute`/`Recurring` profiles without `startSchedule` fields are rejected.                    |
 | K01.FR.41 | ✅     | `Relative` profiles with `startSchedule` fields are rejected.                                   |
 | K01.FR.42 | ⛽️     |                                                                                                 |
-| K01.FR.43 |        |  Open question to OCA - https://oca.causewaynow.com/wg/OCA-TWG/mail/thread/4254                 |
+| K01.FR.43 |        |  Open question to OCA - <https://oca.causewaynow.com/wg/OCA-TWG/mail/thread/4254>                 |
```
Comment (ghost) [inline doc/ocpp_201_status.md]: nit - I do not think the brackets are needed. It is the formatter.

File: tests/lib/ocpp/v201/test_composite_schedule.cpp [hunk 2]
```diff
@@ -813,4 +813,27 @@ TEST_F(ChargepointTestFixtureV201, K08_CalculateCompositeSchedule_ExternalOverri
     ASSERT_EQ(actual, expected);
 }
 
+TEST_F(ChargepointTestFixtureV201, K08_CalculateCompositeSchedule_NoProfiles) {
```
Comment (ghost) [inline tests/lib/ocpp/v201/test_composite_schedule.cpp]: Could the test name mention the expected outcome results?

File: doc/ocpp_201_status.md (hunk 1 above)
Comment (christopher-davis-afs) [reply doc/ocpp_201_status.md]: Fixed :)

File: tests/lib/ocpp/v201/test_composite_schedule.cpp (hunk 2 above)
Comment (christopher-davis-afs) [reply tests/lib/ocpp/v201/test_composite_schedule.cpp]: Renamed to `K08_CalculateCompositeSchedule_NoProfilesAppliesNoLimits`

**Review by ghost — APPROVED** [review]
LGTM - ☕

**Pull request merged**